docs/topics/downloader-middleware.rst

```diff
@@ -922,13 +922,68 @@ RobotsTxtMiddleware
     To make sure Scrapy respects robots.txt make sure the middleware is enabled
     and the :setting:`ROBOTSTXT_OBEY` setting is enabled.
 
+    This middleware has to be combined with a `robots.txt <http://www.robotstxt.org/>`_ parser.
```

Comment:
You have lots of references from `robots.txt` to http://www.robotstxt.org/ all over this page. You could benefit from the first form of external linking here: http://docutils.sourceforge.net/docs/user/rst/quickref.html#external-hyperlink-targets

That way, you only need to specify the target link once at the end of the page.

I’m not sure, but you might need to use:

    `robots.txt`_

instead of just:

    robots.txt_

if the latter does not work, and it might not work because of the period.